contrib/kcp-dex/README.md

```diff
@@ -35,6 +37,41 @@ go run ./cmd/kcp start \
 --oidc-ca-file=127.0.0.1.pem
 ```
 
+## Structured Authentication Config
```

Comment:
This isn't really what I meant. Can you please add content to `/docs/` to make sure that we explain (in general terms, not the dex example) how to use this feature with kcp?

Reply:
Ah, gotcha. That sounds sensible. Can you please open a follow-up issue (probably just a simple placeholder) so we can track that, since the PR will close the original issue?

Reply:
Sure, it makes sense to treat it as seperate issue in this case

Reply:
Issue created under: https://github.com/kcp-dev/kcp/issues/3297